src/rollup/index.ts

```diff
@@ -260,9 +260,10 @@ export default function rollup(
 					// populate asset files into output
 					const assetFileNames = outputOptions.assetFileNames || 'assets/[name]-[hash][extname]';
 					const outputBundle: OutputBundle = graph.finaliseAssets(assetFileNames);
-
 					const inputBase = commondir(
-						chunks.filter(chunk => chunk.entryModule).map(chunk => chunk.entryModule.id)
+						chunks
+							.filter(chunk => chunk.entryModule && isAbsolute(chunk.entryModule.id))
```

Comment:
Doesn't the ID generation for preserve modules now guarantee this is always true?